param/optical_flow_params_1.0.yaml

```diff
@@ -0,0 +1,54 @@
+# Settings for the optical flow estimator
+optical_flow_estimator:
+    # fov of the camera
+    fov: 1.07
```

Comment:
Supposed to be 1.07, the one in the line detector is fixed in a separate PR

Reply:
Since that branch is not going to be merged soon we should be moving to fix this error. Particularly as I plan on testing the grid detector within a day or two. Can we fix it here?